P5/Source/Specs/att.datcat.xml

```diff
@@ -6,52 +6,214 @@ See the file COPYING.txt for details
-      <desc versionDate="2012-04-15" xml:lang="en">contains a PID (persistent identifier) that aligns the content of the given
-        element or the value of the given attribute with the appropriate simple Data
-        Category (or categories) in ISOcat.</desc>
-      <desc xml:lang="ja" versionDate="2019-05-20">指定された要素の内容または指定された属性の値を、ISOcatの適切かつ単純なデータカテゴリ（または複数のデータカテゴリ）と対応付けるPID（永続的識別子）を含む。</desc>
-      <datatype maxOccurs="unbounded"><dataRef key="teidata.pointer"/></datatype>
+    <attDef ident="valueDatcat" usage="opt">
+      <desc versionDate="2022-09-17" xml:lang="en">provides a definition of, and/or general
+        information about a value of an information container (element content or attribute value),
+        by reference to an external taxonomy or ontology. Used especially where a contrast with
+        <att>datcat</att> is needed.</desc>
+      <desc xml:lang="ja" versionDate="2019-05-20"
+        >指定された要素の内容または指定された属性の値を，ISOcatの適切かつ単純なデータカテゴリ（または複数のデータカテゴリ）と対応付けるPID（永続的識別子）を含む．</desc>
+      <datatype maxOccurs="unbounded">
+        <dataRef key="teidata.pointer"/>
+      </datatype>
+    </attDef>
+    <attDef ident="targetDatcat" usage="opt">
+      <desc versionDate="2022-09-17" xml:lang="en">provides a definition of, and/or general
+        information about, information structure of an object referenced or modeled by the
+        containing element, by reference to an external taxonomy or ontology. This attribute has the
+        characteristics of the @datcat attribute, except that it addresses not its containing
+        element, but an object that is being referenced or modeled by its containing element.</desc>
+      <datatype maxOccurs="unbounded">
+        <dataRef key="teidata.pointer"/>
+      </datatype>
     </attDef>
   </attList>
   <exemplum xml:lang="en">
-    <p>In this example <att>dcr:datcat</att> relates the feature name to the data category
-      "partOfSpeech" and <att>dcr:valueDatcat</att> the feature value to the data category
-      "commonNoun". Both these data categories reside in the ISOcat DCR at <ref target="http://www.isocat.org">www.isocat.org</ref>, which is the DCR used by ISO
-      TC37 and hosted by its registration authority, the MPI for Psycholinguistics in
-      Nijmegen.</p>
-    <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#UND">
+    <p>The below example presents the TEI encoding of the <ref
+        target="https://en.wikipedia.org/wiki/Name%E2%80%93value_pair">name-value pair</ref>
+        "<tag>part-of-speech, common-noun</tag>" (where the name/key is abbreviated as "POS",
+      and the value is symbolized by "NN"). The <att>datcat</att> attribute relates the feature <hi
+        rend="italic">name</hi> to the data category "part of speech", while the attribute
+        <att>valueDatcat</att> relates the feature <hi rend="italic">value</hi> to the data category
+      "common noun". Both these data categories should be defined in an external and preferably open
```

Comment:
should they be defined in or point to an external ... taxonomy?

Reply:
DCs -- defined. References -- point. I'll see how I can improve the wording.

Reply:
I'd say it's all good. Will see, maybe split this into two `<p>`s -- but I wouldn't like to omit the info.

Reply:
I have reworded this slightly but I won't be able to see how it reads, because the build won't compile due to the proper reference to the validation script now...